<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportLog</class>
 <widget class="QDialog" name="ImportLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge Log</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../nota.qrc">
    <normaloff>:/img/nb.png</normaloff>:/img/nb.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fromLabel">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="sourceLine"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTextEdit" name="messageBox">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="doneButton">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="statusLine"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../nota.qrc"/>
 </resources>
 <connections/>
</ui>
